all/entrypoint.sh

```diff
@@ -396,6 +396,10 @@ if [[ $GEN_VALIDATOR == true && $GEN_LANG == "gogo" ]]; then
     GEN_STRING="$GEN_STRING --validate_out=lang=gogo${VALIDATOR_SOURCE_RELATIVE}:$OUT_DIR"
 fi
 
+if [[ $GEN_VALIDATOR == true && $GEN_LANG == "java" ]]; then
+    GEN_STRING="$GEN_STRING --validate_out=lang=java${VALIDATOR_SOURCE_RELATIVE}:$OUT_DIR"
```

Comment:
VALIDATOR_SOURCE_RELATIVE is a go/gogo flag, so there is no need to set it.

Reply:
@ido-namely took care of it